tmtools/io.py

```diff
@@ -14,13 +15,52 @@
 import numpy as np
 
 
-def get_structure(fname):
+def get_structure(fname, format="pdb"):
+    """
+    Load and parse a structure file.
+
+    Parameters
+    ----------
+    fname : str
+       Path to the structure file.
+    format : {'pdb', 'mmcif'}, optional
+       Format of the structure file. Default is 'pdb'.
+
+    Returns
+    -------
+    Bio.PDB.Structure.Structure
+       Parsed structure object.
+
+    Raises
+    ------
+    ValueError
+       If an invalid format is provided.
+
+    Notes
+    -----
+    This function uses Biopython's structure parsers to load the file.
+    Warnings are suppressed during parsing.
+
+    Examples
+    --------
+    >>> structure = get_structure('1abc.pdb')
+    >>> structure = get_structure('1xyz.cif', format='mmcif')
+    """
+    # Check if format provided is valid
+    if format not in ["pdb", "mmcif"]:
+        raise ValueError("Invalid structure format")
```

Comment:
Maybe add the format to the error message:

```suggestion
        raise ValueError(f"Invalid structure format: {format!r}")
```

Reply:
Done!